### PR TITLE
[i-24] Encrypt name, description, amount of Expense

### PR DESCRIPTION
**Pull request by vasilismantz**
Using [mongoose-encryption](https://www.npmjs.com/package/mongoose-encryption) package
#24

### CODE REVIEW

File: apps/api/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "jsonwebtoken": "8.5.1",
     "lodash": "4.17.20",
     "mongoose": "5.10.0",
+    "mongoose-encryption": "^2.0.3",
```
Comment (alewis729) [inline apps/api/package.json:36]: Should install a specific version.

File: apps/api/src/models/expenses.js [hunk 2]
```diff
@@ -34,6 +35,13 @@ const expenseSchema = new Schema(
   { timestamps: true }
 );
 
+let secret = process.env.ENC_SECRET;
+
+expenseSchema.plugin(encrypt, {
+  secret: secret,
```
Comment (alewis729) [inline apps/api/src/models/expenses.js:41]:
```suggestion
expenseSchema.plugin(encrypt, {
  secret: process.env.ENC_SECRET,
```

File: apps/api/.env.example [hunk 3]
```diff
@@ -3,3 +3,4 @@ JWT_SECRET=
 MOCKS=false
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
+ENC_SECRET=
```
Comment (alewis729) [inline apps/api/.env.example:6]: What should this value be for a local & for a production environment? Any secret-type string? I also think that `ENCRYPTION_SECRET` is a slightly better name.

**Review by alewis729 — CHANGES_REQUESTED** [review]

File: apps/api/.env.example (hunk 3 above)
Comment (vasilismantz) [reply apps/api/.env.example:6]: According to the [documentation](https://www.npmjs.com/package/mongoose-encryption), the secret should be a 32/64 bit base64 string.

**Review by alewis729 — APPROVED** [review]

**Pull request merged**